Dockerfile

```diff
@@ -8,4 +8,6 @@ RUN go build -o ./openfga ./cmd/openfga
 FROM alpine as final
 EXPOSE 8080
 COPY --from=builder /app/openfga /app/openfga
-ENTRYPOINT ["/app/openfga"]
\ No newline at end of file
+COPY --from=builder /app/static /app/static
+WORKDIR /app
```

Comment:
Do we need this here? And, if so, should we change the entrypoint?

Reply:
The binary uses a relative path for the static assets, so it's important that we set our working directory to the same directory where the static assets are located.

Reply:
I noticed when I tried to run it. 👍